functions/AutoRemediateEC2-003.js

```diff
@@ -26,6 +25,6 @@ module.exports.handler = (event, context, callback) => {
 
   function handleError (message) {
     message = message || 'Failed to process request.'
-    return callback(new Error(message))
+    throw new Error(message)
```

Comment:
🔧 This handler is using `callback` so this shoudn't `throw` an error. I.e. there is no need to change this line